microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -305,21 +305,21 @@ In this step of the tutorial, we'll create a basic temperature unit conversion a
 
     ![Running your new PWA, with the frontend code, on localhost](./index-images/sample-pwa-app-with-frontend-code.png)
 
-Your app does something useful now, but it can't be installed yet, because there's no service worker. You'll make your app installable in the next step, by creating a service worker.
+Your app does something useful now, and it can be installed as a standalone app by users. Before installing the app, create a service worker to make the app work offline.
 
 
 <!-- ====================================================================== -->
 ## Step 5 - Add a service worker
 
-Service workers are a key technology that help make PWAs faster and independent of network conditions.
+Service workers are a key technology that help make PWAs fast and independent of network conditions.
 
 Service workers are specialized [Web Workers](https://developer.mozilla.org/docs/Web/API/Web_Workers_API) that intercept network requests from your PWA and enable scenarios that were previously limited to native apps, including:
 
 *  Offline support.
 *  Advanced caching.
 *  Running background tasks such as receiving PUSH messages, adding badges to the app icon, or fetching data from a server.
 
-For Microsoft Edge to be able to install the app, your app must have a service worker file.
+Your PWA doesn't need to have a service worker for Microsoft Edge to be able to install the app. However, we recommend adding a service worker to your PWA to make it more reliable and faster.
```

Comment:
temp comment: added "offline":
```
Your PWA doesn't need to have a service worker for Microsoft Edge to be able to install the app.  However, we recommend adding a service worker to your PWA:
* To make it more reliable.
* To make it faster.
* To allow it to work offline.
```
@captainbrosset

Reply:
Offline capability is something that applies to all PWAs that use a service worker, and is, indeed, something we want to emphasize.

That being said, "reliable" and "offline" are a bit of a repeat.

- Reliable: you're in the subway, suddenly lose connectivity, but you can still use the app. It's not that you've chosen to be completely offline (like when using airplane mode), but the environment you're in doesn't provide a consistent internet connection. So the app, at times, think it's offline and the service worker makes the app continue to work.
- Offline: you've chosen to be offline for whatever reason, but the app continues to function, thanks to the service worker.

We can either keep them separate (like in your proposal), or merge them somehow, like "To make it more reliable, such as when your device is not consistently connected to the internet, or when it is offline".

Reply:
will try combined-item wording

Reply:
```
A PWA doesn't need to have a service worker for Microsoft Edge 
to be able to install the app.  However, we recommend adding a 
service worker to your PWA to make it faster, and to make your 
PWA more reliable, such as when your device has an intermittent 
network connection or is offline.
```